playwright/tests/screenshots/browsers.spec.ts

```diff
@@ -17,22 +17,25 @@ test.describe('Browsers', () => {
     const dialog = browser.dialog;
 
     await browser.openTab('Attribute');
-    await expect(browser.table.getByRole('row', { name: 'addPerson' }).first()).toBeVisible();
+    await browser.table.getByRole('row', { name: 'addPerson' }).first().click();
+    await expect(browser.help).toContainText('in.addPerson');
     await screenshot(dialog, 'browser-attribute.png');
 
     await browser.openTab('CMS');
-    await browser.search('Customer');
-    await expect(browser.table.getByRole('row', { name: 'Customer' })).toBeVisible();
+    await browser.search('subject');
+    await browser.table.getByRole('row', { name: 'subject' }).click();
+    await expect(browser.help).toContainText('Welcome');
     await screenshot(dialog, 'browser-cms.png');
 
     await browser.openTab('Function');
     await browser.search('role');
-    await expect(browser.table.getByRole('row', { name: 'ivy' })).toBeVisible();
+    await browser.table.getByRole('row', { name: 'security :' }).click();
+    await expect(browser.help).toContainText('ivy.security');
     await screenshot(dialog, 'browser-function.png');
 
     await browser.openTab('Type');
     await browser.search('Per');
-    await expect(browser.table.getByRole('cell', { name: 'Person :' })).toBeVisible();
+    await browser.table.getByRole('row', { name: 'Person :' }).first().click();
```

Comment:
couldn't display the "help" here in a predictable manor. 
first, there is no real addition ... the help just shows the select value ... here "Person". I would have expected it to at least show the DataClass description. Not very important but seems an easy one to improve on the backend.
second, asserting it to be present works on my machine ... not on the CI.

Reply:
Yes, it is the case that the help doesn't really help much with the typebrowse and only shows the name of the type for the moment. This could of course be improved. I think for the moment, a screenshot of the help only makes sense for the CMS & Functionbrowser.

Unfortunately I don't see why the assertion doesn't work. Maybe @ivy-lli has an idea.

Reply:
> Unfortunately I don't see why the assertion doesn't work. Maybe @ivy-lli has an idea.

Not really important for me now; I see that the clicked column becomes kind of un-selected after clicking it. Could surely be solved re-clicking. But I think it could be a pointer to a real problem that exits... we may stumble over this again when streamlining the browsers :shrug: